Extract short covid FSH out of existing long one

oreinecke opened the pull request:
Hi,

I noticed that the FSH file for the short COVID survey was still missing, and Lisa assured me that other pilots would be using that questionnaire as well. I think it would be nice if the questionnaire appeared on the GK FHIR IG as well.

So I used the existing long COVID survey and removed all the sections that aren't in the Word draft of the short survey. Some of the inlined ValueSets at the top aren't used in the short survey anymore, so I removed them as well. I've kept the structure with all the wrapper group items.

I am not sure wether to recount all the prefixes or just remove them. Either way works fine for me. Feel free to change the title or instance properties.

oreinecke committed:
Extract short covid FSH out of existing long one

otiliakocsis reviewed (commented):
Hi Oscar,

Yes, the short one is part of the questionnaire for all languages. Thank you for preparing this one. I am fine with it, so if Giorgio approves it, it should be considered final.

br,
Otilia

gcangioli commented:
The build fails due to a name duplication in the fsh files

- error Skipping Instance: an Instance named inlineValueSet-covid19-4-1 already exists.  
- error Skipping Instance: an Instance named inlineValueSet-covid19-7-7 already exists.

oreinecke commented:
Sorry about that, I am not at all familiar with FSH. It seems that the `questionnaireCOVIDshort.fsh` could simply reuse the instances from `questionnaireCovid.fsh`, so I'll drop lines 2-25. Sounds good?

oreinecke committed:
squash! Extract short covid FSH out of existing long one

(remove duplicate instances inlineValueSet-covid19-4-1 and
inlineValueSet-covid19-7-7)
